app/models/vm.rb

```diff
@@ -4,6 +4,7 @@ class Vm < VmOrTemplate
   has_one :container_deployment_node
 
   virtual_has_one :supported_consoles, :class_name => "Hash"
+  virtual_column :transformed, :type => :boolean
```

Comment:
Do we still need to expose this method?  Can't the policy be written to check for the tag directly?